src/Hit/Git.hs

```diff
@@ -72,6 +73,18 @@ runCommit (T.strip -> msg) (not -> hasIssue)
            let issue = "#" <> show n
            in "[" <> issue <> "] " <> msg <> "\n\nResolves " <> issue
 
+-- / @hit fix@ command
+runFix :: Maybe Text -> IO ()
+runFix = \case
+    Nothing -> do
+        "git" ["add", "."]
+        "git" ["commit", "-m", "Fix after review"]
```

Comment:
I guess you can find [`fromMaybe`](https://hackage.haskell.org/package/relude-0.4.0/docs/Relude-Monad-Reexport.html#v:fromMaybe) useful in here. With it, you won't need to pattern match on the `Maybe Text` :slightly_smiling_face: